airmozilla/popcorn/static/popcorn/PopcornEditor/src/editor/media-gallery-editor.js

```diff
@@ -700,6 +702,16 @@ define( [ "util/lang", "util/uri", "util/xhr", "util/keys", "util/mediatypes", "
     _searchInput.addEventListener( "keydown", onEnter );
 
     _addBtn.addEventListener( "click", onAddMediaClick );
+
+    Event.listen('load', function (data) {
+      var clipData = data.data.media[0].clipData,
+          clip;
+
+      for (url in clipData) {
```

Comment:
dang it! Good catch. Perhaps put in a comment that explains that for the next person who has the same thought as me.